src/components/OrderType/OrderType.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactElement, useState } from 'react';
+
+export const OrderType = (): ReactElement => {
+  const [activeButton, setActiveButton] = useState<number>(1);
+
+  return (
+    <div>
+      <h1 className="font-poppins mt-2 text-center text-xl font-semibold">Order Summary</h1>
+      <div className="mt-3 mb-4 grid h-8 grid-cols-2 rounded-[14px] bg-gray-200 align-bottom">
+        <div className="col-span-1 ml-1 pt-1">
+          <button
+            onClick={() => setActiveButton(1)}
+            className={`font-poppins rounded-xl ${activeButton == 1 ? 'bg-[#ffffff]' : ''} px-4 text-[#333F4C] hover:cursor-pointer hover:bg-gray-50`}
```

Comment:
If you rebase, font-poppins should be applied everywhere now so no need to use it again and again in classes. Would only need to change font weight and style individually.